### PR TITLE
have user say how many people will be on call and create obj in db

### PR DESCRIPTION
**Pull request by tosmith97**

### CODE REVIEW

File: app/calls/views.py [hunk 1]
```diff
@@ -58,13 +58,21 @@ def call_events():
     if url:
         print('we have url')
         response = client.get_recording(url)
-        fn = os.path.join(*[os.getcwd(), 'recordings', res.get('recording_uuid', datetime.today().strftime('%Y-%m-%d')) + '.wav'])
-        a = AudioSegment.from_file(BytesIO(response), channels=2, sample_width=2, frame_rate=16000)
+        uuid = res.get('recording_uuid', datetime.today().strftime('%Y-%m-%d'))
+        fn = os.path.join(*[os.getcwd(), 'recordings', uuid + '.wav'])
+
+        # getting most recent is bad but for practical purposes works
```
Comment (jsinge97) [inline app/calls/views.py]: Doesn't the post request include the uuid of call that we should store in db?

File: app/calls/views.py [hunk 2]
```diff
@@ -76,5 +84,9 @@ def call_recordings():
 
 @calls.route('/create-call/new', methods=['GET', 'POST'])
 def new_call():
+    n_channels = request.n_channels
```
Comment (jsinge97) [inline app/calls/views.py]: are we storing the phone numbers here? It just seems like number of channels

File: app/calls/views.py [hunk 3]
```diff
@@ -41,7 +41,7 @@
 def start_call():
     request_data = json.loads(request.data)
     ncco_data = {}
-    ncco_data['hostname'] = 'http://8978f380.ngrok.io' #current_app.config['HOST_NAME']
+    ncco_data['hostname'] = '8978f380.ngrok.io' #current_app.config['HOST_NAME']
```
Comment (jsinge97) [inline app/calls/views.py]: Don't hard code this

File: app/calls/views.py [hunk 4]
```diff
@@ -42,29 +42,34 @@ def start_call():
...
     print(request_data)
     print(ncco)
     sys.stdout.flush()
 
-    # attack uuid to call db object
+    # attach uuid to call db object
     initial_phone_number = request_data.get('from', None)
     print('number is %s' % initial_phone_number)
     if initial_phone_number:
         # get most recent call based on who called the number + hasn't been updated before + most recent
```
Comment (jsinge97) [inline app/calls/views.py]: were we able to retrieve the call info from events??

File: app/calls/views.py [hunk 5]
```diff
@@ -51,35 +51,32 @@ def start_call():
...
-    print(ncco)
-    sys.stdout.flush()
 
     # attach uuid to call db object
     initial_phone_number = request_data.get('from', None)
     print('number is %s' % initial_phone_number)
     if initial_phone_number:
         # get most recent call based on who called the number + hasn't been updated before + most recent
-        last_call = db.session.query(Call).filter(Call.initial_phone_number == initial_phone_number, Call.call_uuid == None).order_by(Call.id.desc()).first()
+        last_call = db.session.query(Call).filter(Call.initial_phone_number == initial_phone_number).order_by(Call.id.desc()).first()
```
Comment (jsinge97) [inline app/calls/views.py]: better

File: app/calls/views.py (hunk 4 above)
Comment (jsinge97) [reply app/calls/views.py]: https://stackoverflow.com/questions/44783656/subprocess-stdout-into-a-variable-for-ngrok

File: app/calls/views.py (hunk 3 above)
Comment (tosmith97) [reply app/calls/views.py]: it will be hardcoded for now

**Pull request merged**